dask/dataframe/groupby.py

```diff
@@ -1765,7 +1765,6 @@ def aggregate(self, arg, split_every=None, split_out=1, shuffle=None):
             chunk=_groupby_apply_funcs,
             chunk_kwargs=dict(
                 funcs=chunk_funcs,
-                sort=self.sort,
```

Comment:
There is technically nothing **wrong** with this line, but it adds little-to-no benefit, and happened to break a fragile test in dask-cudf. Therefore, I'd prefer to remove it for now.

Reply:
So strong thoughts from this on me. Could we update the `dask-cudf` test to be less fragile?

Reply:
>Could we update the dask-cudf test to be less fragile?

That is certainly the plan - I am working on a dask-cudf groupby overhaul that will change the test anyway.